src/pages/documentation/adobe-analytics/api-reference.md

```diff
@@ -149,17 +121,9 @@ Retrieves the Analytics tracking identifier that is generated for this app/devic
 
 Starting with v1.2.9 (Android) / v3.0.3(iOS AEPAnalytics) / v2.5.1 (iOS ACPAnalytics) this API does not generate or retrieve a new tracking identifier (AID) for new visitors. For the visitors which have an AID previously generated will continue retrieve the AID value with this API, and new users will use the ECID (MID) value as the primary identity.<br/><br/>Before using this API, see the documentation on identifying [unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
```

Comment:
Update for "Starting with ... " let's remove the v3.0.3 reference from the acp repo